### PR TITLE
fix: wrong artwork image flashing onScoll

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [ONYX-653] 

### Description

This PR solves the following issue with the new works for you screen: When the user scrolls down, and starts seeing new artwork images, the image flashes and a wrong ones shows up initially. 

The fix, since this screen has no pagination logic and it only shows a small list of artworks, we can avoid this by using RN default `Flatlist` here.

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- NWFY: fix wrong artwork image flashing onScoll - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-653]: https://artsyproduct.atlassian.net/browse/ONYX-653?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/NewWorksForYou/Components/NewWorksForYouList.tsx [hunk 1]
```diff
@@ -79,7 +77,7 @@ export const NewWorksForYouList: React.FC<NewWorksForYouListProps> = ({ viewer }
   )
 }
 
-const AnimatedFlashlist = Animated.createAnimatedComponent(FlashList) as unknown as typeof FlashList
+// const AnimatedFlashlist = Animated.createAnimatedComponent(FlashList) as unknown as typeof FlashList
```
Comment (gkartalis) [inline src/app/Scenes/NewWorksForYou/Components/NewWorksForYouList.tsx:80]: lets get rid of this?

**Review by gkartalis — APPROVED** [review]
nice!

File: src/app/Scenes/NewWorksForYou/Components/NewWorksForYouList.tsx (hunk 1 above)
Comment (gkartalis) [reply src/app/Scenes/NewWorksForYou/Components/NewWorksForYouList.tsx:80]:
```suggestion
```

Comment (MounirDhahri) [reply src/app/Scenes/NewWorksForYou/Components/NewWorksForYouList.tsx:80]: Ouff sorry i didn't check thoroughly, I will get rid of it.

**Pull request merged**